bugzilla/base.py

```diff
@@ -1131,20 +1134,52 @@ def getbug(self, objid,
             extra_fields=extra_fields)
         return Bug(self, dict=data, autorefresh=self.bug_autorefresh)
```

Comment:
Let's use your new private method here, too.

```suggestion
        return self._to_bug(bug_data=data, autorefresh=self.bug_autorefresh)
```

Reply:
Yeah, this _to_bug cleanup is nice. If it was a separate patch I'd apply it